graphql/src/main/java/io/strimzi/admin/graphql/GraphQLService.java

```diff
@@ -0,0 +1,120 @@
+package io.strimzi.admin.graphql;
+
+import graphql.GraphQL;
+import graphql.schema.GraphQLSchema;
+import graphql.schema.idl.SchemaGenerator;
+import graphql.schema.idl.SchemaParser;
+import graphql.schema.idl.TypeDefinitionRegistry;
+import io.strimzi.admin.graphql.registration.GraphQLRegistration;
+import io.strimzi.admin.graphql.registration.GraphQLRegistrationDescriptor;
+import io.strimzi.admin.graphql.registration.RuntimeWiringRegistry;
+import io.strimzi.admin.http.server.registration.RouteRegistration;
+import io.strimzi.admin.http.server.registration.RouteRegistrationDescriptor;
+import io.vertx.core.CompositeFuture;
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.handler.graphql.GraphQLHandler;
+import io.vertx.ext.web.handler.graphql.GraphiQLHandler;
+import io.vertx.ext.web.handler.graphql.GraphiQLHandlerOptions;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.ServiceLoader;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class GraphQLService implements RouteRegistration {
```

Comment:
A line of Javadoc summarizing what this class does in a sentence or two.

Reply:
Added javadoc